src/day6.scala

```diff
@@ -0,0 +1,81 @@
+// using scala 3.1.0
+
+import scala.util.Using
+import scala.collection.mutable
+import scala.io.Source
+
+@main def part1(): Unit =
+  println(s"The solution is ${part1(readInput())}")
+
+@main def part2(): Unit =
+  println(s"The solution is ${part2(readInput())}")
+
+def readInput(): String =
+  Using.resource(Source.fromFile("input/day6"))(_.mkString.trim)
+
+// "Find a way to simulate lanternfish. How many lanternfish would there be after 80
+// days?"
+def part1(input: String): Int =
+  simulate(
+    days = 80,
+    initialPopulation = Fish.parseSeveral(input)
+  )
+
+// "You can model each fish as a single number that represents the number of days
+// until it creates a new lanternfish."
+case class Fish(timer: Int)
+
+object Fish:
+  // "Suppose you were given the following list:
+  //
+  // 3,4,3,1,2
+  //
+  // This list means that the first fish has an internal timer of 3, the second fish
+  // has an internal timer of 4, and so on until the fifth fish, which has an
+  // internal timer of 2."
+  def parseSeveral(input: String): Seq[Fish] =
+    for timerStr <- input.split(",")
```

Comment:
There is a warning on this line:
```
[warn] .\src/day6.scala:39:5: method copyArrayToImmutableIndexedSeq in class LowPriorityImplicits2 is deprecated since 2.13.0: Implicit conversions from Array to immutable.IndexedSeq are implemented by copying; Use the more efficient non-copying ArraySeq.unsafeWrapArray or an explicit toIndexedSeq call
[warn]     for timerStr <- input.split(",")
[warn]     ^
```
It should be addressed.

Reply:
How did you get the warning? Scala-cli does not complain.

Reply:
There is a known bug in Bloop that could explain why scala-cli does not complain. You should see it if you do `scala-cli clean .` then `scala-cli compile .`.

Reply:
It was scala-cli that complained for me. Latest scala-cli, on Windows.

Reply:
OK, funny, I did run `scala-cli clean src/day6.scala` but the warning did not show up. However, after running `scala-cli clean src/` then I can see the warning…

Reply:
Thank you guys. This is fixed in 46e4344